pmm-app/src/pmm-qan/panel/provider/provider.tsx

```diff
@@ -186,6 +186,7 @@ export const UrlParametersProvider = ({ timeRange, children }) => {
     } else {
       newState.pageNumber = 1;
       delete newState.queryId;
+      // @ts-expect-error
```

Comment:
just out of curiosity: why does this line cause an error?

Reply:
Because `querySelected` can be `undefined` and it says it cannot delete something that can be `undefined` (although in our logic I don't think it will ever be `undefined`)